udp_server.py

```diff
@@ -1,31 +1,48 @@
 # Import the socket library
 import socket
 
-buffer_size  = 1024
-
-# Get the paramters from the user
-###### NEED TO ADD MANY CHECKS #######
-print("What is the IP that we will listen to?")
-local_ip = input()
-print("What is the port?")
-local_port = int(input())
-
-# The DGRAM socket corresponds to the UDP
-udp_server_socket = socket.socket(family=socket.AF_INET, type=socket.SOCK_DGRAM)
-
-# Bind to address and ip
-udp_server_socket.bind((local_ip, local_port))
-
-print("UDP server up and listening")
-
-# Listen for incoming DGRAM
-while(True):
-    bytes_address_pair = udp_server_socket.recvfrom(buffer_size)
-    message = bytes_address_pair[0]
-    address = bytes_address_pair[1]
-    client_msg = "Message from Client:{}".format(message)
-    client_ip  = "Client IP Address:{}".format(address)
-    print(client_msg)
-    print(client_ip)
-    # Sending a reply that is equal to what we received to the client
-    udp_server_socket.sendto(message, address)
+# Constants
+BUFFER_SIZE  = 1024
+
+
+# Function gets the ip address from the machine, needs to be given a port manually
+def getAddressPort():
+	##### Add script argument for getting the ip and port, enforcing int ##### 
+	print("Enter the server's ip:")
+	local_ip = input()
+	print("What is the port?")
+	local_port = int(input())
+	return [local_ip, local_port]
+
+
+def main():
+	# The DGRAM socket corresponds to the UDP
+	udp_server_socket = socket.socket(family=socket.AF_INET, type=socket.SOCK_DGRAM)
+
+	# Get ip and port
+	local_ip, local_port = getAddressPort()
+
+	# Bind to address and ip
+	udp_server_socket.bind((local_ip, local_port))
+	print("UDP server up and listening")
+
+	message = b""
```

Comment:
Forgot to change it yesterday.
It's in binary, so it also has that 'b' at the beginning of the string.